tests/test_otel_logs.py

```diff
@@ -0,0 +1,107 @@
+from __future__ import annotations
+
+import json
+from typing import Any
+
+from dirty_equals import IsInt, IsStr
+from inline_snapshot import snapshot
+from opentelemetry._events import Event, get_event_logger, get_event_logger_provider
+from opentelemetry._logs import LogRecord, SeverityNumber, get_logger, get_logger_provider
+from opentelemetry.sdk._logs.export import InMemoryLogExporter, SimpleLogRecordProcessor
+from opentelemetry.sdk.resources import Resource
+
+import logfire
+
+
+def test_otel_logs_supress_scopes(logs_exporter: InMemoryLogExporter, config_kwargs: dict[str, Any]) -> None:
+    record = LogRecord(
+        timestamp=1,
+        observed_timestamp=2,
+        trace_id=3,
+        span_id=4,
+        severity_text='INFO',
+        severity_number=SeverityNumber.INFO,
+        body='body',
+        attributes={'key': 'value'},
+    )
+
+    logfire.suppress_scopes('scope1')
```

Comment:
I guess this method already exists, so not for this PR, but may I suggest we somehow allow restricting scopes by level? E.g. "only allow error or above for scope1, allow info or above for scope2"

Reply:
It's been requested and considered, just not prioritized.